war/src/main/less/modules/side-panel-tasks.less

```diff
@@ -59,6 +59,32 @@
     }
   }
 
+  .task-icon-badge {
+    position: relative;
+    color: currentColor;
+    letter-spacing: 0;
+    z-index: 0;
+    margin-left: auto;
+    min-width: 24px;
+    text-align: center;
+    padding: 0 8px;
+    line-height: 0.75rem;
+    font-size: 0.75rem;
+
+    &::before {
+      content: "";
+      position: absolute;
+      inset: -6px 0;
+      background: var(--item-background--hover);
+      border-radius: 100px;
+      z-index: -1;
+    }
+  }
+
+  .task-link-text {
+    display: contents;
+  }
+
   .task-link-text {
     display: contents;
   }
```

Comment:
Improperly resolved conflict, or what is this for?

Reply:
You're correct 👍 I've removed it as part of #7084